compiler/qsc_codegen/src/qir/rir_utils.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: this module could be renamed to `rir_building.rs` or `rir_builder.rs`. I think it describes a bit better what it is used for.